addons/mail/static/src/models/thread/thread.js

```diff
@@ -1106,7 +1123,7 @@ function factory(dependencies) {
          * @returns {mail.message}
          */
         _computeLastCurrentPartnerMessageSeenByEveryone() {
-            if (!this.partnerSeenInfos || !this.orderedMessages) {
+            if (!this.partnerSeenInfos || !this.orderedNonTransientMessages) {
```

Comment:
I think this condition was/still is incorrect. `orderedMessages` & `orderedNonTransientMessages` are arrays so they will never be false.

Reply:
Same for `partnerSeenInfos`. I guess it was to prevent crash depending on compute order at a time where relational fields were not array initially?